### DISCUSSION TITLE
分享

### DISCUSSION THREAD

**Discussion by A3148475824** [xiaye13579/BBLL#442] [General]
希望开发者能创造分享功能！BLBL在我的平板上使用！但是我本人喜欢下载视频到本地观看，不论是seal还是bilibilias以及BVD还是忻视Down都需要链接才能下载！如果BLBL不能拥有分享功能或者复制链接功能这软件对我而言没太大用处，TV使用倒是不错！

**Comment by digitalbaka** [comment]
支持你！我也希望。


用人不宜刻，刻则思效者去；交友不宜滥，滥则贡谀者来。
我的个人空间bilibili
在2023年03月12日 23:47，A3148475824 写道：

希望开发者能创造分享功能！BLBL在我的平板上使用！但是我本人喜欢下载视频到本地观看，不论是seal还是bilibilias以及BVD还是忻视Down都需要链接才能下载！如果BLBL不能拥有分享功能或者复制链接功能这软件对我而言没太大用处，TV使用倒是不错！

**Comment by ghost** [comment]
分享功能就没必要了吧，毕竟大部分都是在电视上用的

**Reply by A3148475824** [reply]
我在平板上也在用！手机上在用biliyou和bilihub
